### PR TITLE
FRONTEND: HIV  Black women

### PR DESCRIPTION
**Pull request by eriwarr**
## Description

This PR adds visualizations for HIV deaths, diagnoses, and prevalence in Black women. 

- Updates the `MetricConfig.ts`, `HivProvider.ts`, and `MadLib.ts` with new variables.
-  Sidebar only shows the 'Age' option when viewing HIV data for Black women (on single and compare mode). 
- Updates the CSV files for state-level HIV deaths in Black women as they previously contained diagnosis data. 

## Motivation and Context

Backend PR for adding HIV for Black women has already merged. 

## Has this been tested? How?
Tested locally.

## Screenshots (if appropriate):

Screenshot 2023-05-10 at 3 08 23 PM

https://github.com/SatcherInstitute/health-equity-tracker/assets/72993442/8b491a72-d528-473b-ae6d-e9e84860fa1e

## Types of changes
- New content or feature

## Post-merge TODO
I have inspected frontend changes and run affected data pipelines:
- [ ] on DEV
- [ ] on PROD

## Preview link below in Netlify comment 😎

### CODE REVIEW

**Review by kccrtv — APPROVED** [review]
👍🏻

**Comment by benhammondmusic** [comment]
I'd like to swap out our zero color for An extremely Light purple; the current zero doesn't look great against this color scheme I think

**Comment by benhammondmusic** [comment]
Also I think We should only use the pink color scheme for the intersectional black women topics, not all hiv topics

File: frontend/src/reports/OneVariableReport.tsx [hunk 1]
```diff
@@ -71,15 +72,17 @@ export function OneVariableReport(props: OneVariableReportProps) {
       : null
   )
 
-  const setVariableConfigWithParam = (v: VariableConfig) => {
+  const isRaceBySex = variableConfig?.variableId.includes(BLACK_WOMEN)
```
Comment (benhammondmusic) [inline frontend/src/reports/OneVariableReport.tsx:75]: This is good for now; I think there really be a utility function that includes the logic here: https://github.com/SatcherInstitute/health-equity-tracker/blob/67f955f49977216758e6049632301c18e77f86ca/frontend/src/data/utils/datasetutils.ts#L165

i'd like the functionality to basically look at the metric and determine which special demographic type map to send into the SimpleSelect for demo

**Review by benhammondmusic — APPROVED** [review]
Looks great! I was confused because your screenshots and vids were purple but it looks like you've held off on the color changes for now waiting for an email reply?

**Pull request merged**